presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -283,6 +283,17 @@ Spilling Properties
     this property defines the threshold at which to trigger spilling for a task.  This property
     is ignored for any other spilling strategy.
 
+``experimental.max-revocable-memory-per-node``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    * **Type:** ``data size``
+    * **Default value:** ``16GB``
+
+    This is the amount of memory that is being used by a query that can be spilled to disk.
```

Comment:
This description needs to be reworked. IMO, we do not need to explain revocable memory semantics here.
Those are explained in detail in its own docs

Please consider replacing the description with below one liner
```
This property defines the amount of revocable memory a query can use on each node
```

Reply:
Modified